errors/errors.go

```diff
@@ -57,6 +57,12 @@ var (
 	// ErrInvalidAmount stands for invalid amount of whatever
 	ErrInvalidAmount = Register(13, "invalid amount")
 
+	// ErrInvalidInput stands for general input problems indication
+	ErrInvalidInput = Register(14, "invalid input")
+
+	// ErrExists already exists error
```

Comment:
What is the distinction between ErrExists and ErrDuplicate?
I guess they could be merged